multimod/internal/common/tools.go

```diff
@@ -61,7 +61,7 @@ func GetModuleSet(modSetName, versioningFilename string) (ModuleSet, error) {
 
 // updateGoModVersions updates one go.mod file, given by modFilePath, by updating all modules listed in
 // newModPaths to use the newVersion given.
```

Comment:
Comment needs to be updated to reflect the change.

Reply:
Fixed, thanks!